File: fontes/c/micro/micro03.c
#include<stdio.h>

int main()
{
 int numero;

 printf("Digite um número: ");
 scanf("%d", &numero);
 if (numero >= 100)
 {
 	if (numero <= 200)
 	{
 		printf("O número está no intervalo entre 100 e 200\n");
 	}
 	else
 	{
 		printf("O número não está no intervalo entre 100 e 200\n");
 	}
 }
 else
 {
 	printf("O número não está no intervalo entre 100 e 200\n");
 }
 return 0;
}

File: fontes/pascal/micro/micro03.pas
program Micro03;
    
var 
	numero: INTEGER;
begin
    writeln('Digite um numero: ');
    readln(numero);
    if numero>=100 then
        if numero<=200 then
            writeln('O número está no intervalo entre 100 e 200')
        else
            writeln('O número não está no intervalo entre 100 e 200')
    else
        writeln('O número não está no intervalo entre 100 e 200')
end.
